### PR TITLE
Replacing the quota limit to "Unlimited"

### PR DESCRIPTION
**Pull request by anusapan**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

File: src/command_modules/azure-cli-iot/HISTORY.rst [hunk 1]
```diff
@@ -4,7 +4,8 @@ Release History
 ===============
 0.3.9
 +++++
-* Add support to list all keys 
+* Add support to list all keys
+* Minor fixes
```
Comment (tjprescott) [inline src/command_modules/azure-cli-iot/HISTORY.rst]: Minor fixes should be removed.

Comment (anusapan) [reply src/command_modules/azure-cli-iot/HISTORY.rst]: removed

**Comment by marstr** [comment]
Azure.azure-cli is an exact clone of azcli-CI, just in a different ADO instance. For that reason, it is safe for us to use administrative privilege to bypass this unstarted check.

**Pull request merged**